### ISSUE TITLE
Splashscreen Blank on iOS

### ISSUE THREAD

**Issue by theZaX** [quasarframework/quasar#9501]
**Describe the bug**
The splash screen is a blank screen when a PWA is installed on iOS

**To Reproduce**
go to quasar.dev on an iPhone. Add it to your home screen. The splash screen is blank. The behavior is the same for apps generated using the CLI. The splash screen is correct on android.

**Expected behavior**
The splash screen for apple devices includes the icons specified in the icons folder of the CLI dist output.

**Platform (please complete the following information):**
Quasar Version:
@quasar/app Version:
Quasar mode:

  - [ ] SPA
  - [ ] SSR
  - [X] PWA
  - [ ] Electron
  - [ ] Cordova
  - [ ] Capacitor
  - [ ] BEX

Tested on:
  - [ ] SPA
  - [ ] SSR
  - [X] PWA
  - [ ] Electron
  - [ ] Cordova
  - [ ] Capacitor
  - [ ] BEX

iOS: shows blank screen
Android: works correctly

**Comment by rstoenescu** [comment]
What iPhone are you using?
iOS is pretty picky when it comes to the PWA splashscreen. It needs a graphics file for each of the phones with exact screen resolution (including the `@2`/`@3` density suffixes). We'll do an update on IconGenie for the newer iPhones.

**Comment by theZaX** [comment]
I am using an iPhone 12

**Comment by rstoenescu** [comment]
Please use our `@quasar/icongenie` CLI (v2.4.0 - just released it now) to generate all the necessary splashscreens.
https://quasar.dev/icongenie/introduction

**Comment by theZaX** [comment]
That sounds great it seems to have been fixed on the quasar site too. Do I need to update the quasar cli to make it insert the icons. I updated the icongenie cli and it seems to have generated the correct icons but my index doesn't reference the new icons when I build. The default quasar build without generating icons doesn't seem to work either.

Update: 
I upgraded the cli and it looks like your fix was only added to v3. Is v3 compatible with v1 apps? My app was written in v1 and I'm not yet ready to upgrade.
